flutter/CHANGELOG.md

```diff
@@ -1 +1 @@
-## [1.1.82] - Complete change log present here https://github.com/microsoft/fluentui-system-icons/tags
\ No newline at end of file
+## [1.1.83] - Complete change log present here https://github.com/microsoft/fluentui-system-icons/tags
```

Comment:
This should remain as 82? Because the publish script will auto increment and commit it.

Reply:
@willhou : I used that for mock testing if the auto publish was successful, can we publish with 84 ?

Reply:
No we cannot custom a version here, this needs to match what the CI's run count is, so that it stays in sync with the Android and iOS libraries versions.

Reply:
I can change it back to 82, but since pub.dev already has 83, (since I did that manually to test if it doesn't fail there), we might have to bump the version 84, so it publishes the changes there.

Reply:
Can you remove 83 from pub.dev? We should have the CI publish 83 to pub.dev

Reply:
Is it possible to do that by using this downgrade command? https://dart.dev/tools/pub/cmd/pub-downgrade

Reply:
@willhou : It is not possible to downgrade the package once it is published. The given link is used to downgrade the dependencies used inside a particular package :/